tools/data_source/upload.py

```diff
@@ -303,7 +317,7 @@ def add_file(dataset, registry, json_file, output_path):
             if e.errno != errno.EACCES:
                 raise
     elif link_data_only == 'copy_files':
-        if purge_source and not run_as_real_user:
+        if purge_source:
```

Comment:
The comment below is already addressed with the new comment above `purge_source`, line 86-87, I think it's better to remove the comment.